wiki/Contests/M2P/en.md

```diff
@@ -0,0 +1,103 @@
+---
+tags:
+  - m2p
+  - m2p#1
+  - m2p#2
+  - m2p#3
+  - m2p 1
+  - m2p 2
+  - m2p 3
+  - m2p: taiko
+---
+
+# Mapping to Pander: Taiko
+
+![contest banner](img/banner.jpg "Mapping to Pander: Taiko")
+
+**Mapping to Pander: Taiko** (*M2P*) is an osu!taiko mapping contest series with two primary features: there is only one judge per iteration, and contestants may map any song.
+
+The goal this contest is simple: Contestants must create a map that panders best to the chosen judge for the contest. The judge will not be given any criteria; it will be up to them how they score and rank the entries, either by making their own criteria, or simply sorting them from best to worst.
+
+Each iteration of the contest will have a different judge, with different tastes and criteria from the last.
+
+## Schedules
+
+Iterations usually happen every 1 or 2 months, with each iteration's schedule going as follows:
```

Comment:
it's not clear to me and possibly other readers how many iterations there are going to be. even if it's a "let's figure it out as we go" type of contest, better state this explicitly

Reply:
clarified it a bit more